flink-clients/src/main/java/org/apache/flink/client/program/PackagedProgram.java

```diff
@@ -451,6 +476,31 @@ public ClassLoader getUserCodeClassLoader() {
 		if (jarFile != null) {
 			libs.add(jarFile);
 		}
+
+		if (isPython) {
```

Comment:
We should not put `flink-table` to the class path by default for two reasons:
1) It's not necessary for Python API, as we may support other Python API such as DataStream API in the future. `flink-table` is not necessary for these kinds of Python API
2) It should be done by users manually.

Reply:
Yes, you are right.